cli/args.go

```diff
@@ -210,6 +210,10 @@ func parseTerragruntOptionsFromArgs(terragruntVersion string, args []string, wri
 	opts.Check = parseBooleanArg(args, OPT_TERRAGRUNT_CHECK, os.Getenv("TERRAGRUNT_CHECK") == "true")
 	opts.HclFile = filepath.ToSlash(terragruntHclFilePath)
 	opts.AwsProviderPatchOverrides = awsProviderPatchOverrides
+	opts.JSONOut, err = parseStringArg(args, OPT_TERRAGRUNT_JSON_OUT, "terragrunt_rendered.json")
```

Comment:
Super NIT, but would `JsonOut` be the more Go idiomatic var name?

Reply:
Go recommends using [consistent casing for acronyms](https://github.com/golang/go/wiki/CodeReviewComments#initialisms), and since JSON is an acronym (JavaScript Object Notation), I think this is more idiomatic.

Reply:
Roger